index.html

```diff
@@ -9,6 +9,20 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <div class="menu hidden">
+      <div class="close-btn-wrapper">
+        <button class="close-btn">
+          <img src="./imgs/close-btn.svg" alt="close button" />
+        </button>

```

Comment:
- [ ] You did a great job implementing the requested close button icon! ✨ However, looks like you placed the icon in the wrong place. Kindly remember to stick as much as possible to the Figma design as stated in the requirements.

**Your design:** 
![your-design](https://user-images.githubusercontent.com/43172164/197297369-bb5c7f0e-caf3-4737-9886-fd0b3d81d025.png)

**Figma design:** 
![mobile-menu](https://user-images.githubusercontent.com/43172164/197297870-f5c585ca-3e0b-4ae1-8b14-c0b5416a13ff.png)


- Kindly fix this 🔧